Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidStaticClassesAnalyzer.cs

```diff
@@ -100,15 +97,12 @@ public void Analyze(SyntaxNodeAnalysisContext context)
 
 			// Check if this is an extension class
 			var model = context.SemanticModel.GetDeclaredSymbol(classDeclarationSyntax);
-			if (model is {MightContainExtensionMethods: true})
-			{
-				if (!model.GetMembers().Any(m =>
+			if (model is { MightContainExtensionMethods: true } && !model.GetMembers().Any(m =>
```

Comment:
In complex condition like this, I prefer to create a named method out of the lambda. This reduces the clutter considerably.

Reply:
done